dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -34,10 +34,10 @@
     %p Edit Markdown:
 
     %textarea#level_markdown_textarea
-    #markdown-preview
+    #level_markdown_textarea_preview
 
   .markdown.teacher
     %p Edit Teacher Only Markdown:
 
     %textarea#level_teacher_markdown_textarea
-    #teacher-markdown-preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
Styleguide violation: [We actually require kebab-case for element IDs](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#html).